models/balancer/arbitrum/balancer_v2_arbitrum_liquidity.sql

```diff
@@ -0,0 +1,201 @@
+{{
+    config(
+        schema='balancer_v2_arbitrum',
+        alias='liquidity',
+        unique_key = ['day', 'pool_id', 'token_address'],
```

Comment:
as this is a view, `unique_key` property is not needed. the schema file containing the test for unique combo of columns is sufficient. this would only be used if incremental and performing a merge to join to itself.